traffic_ops/testing/api/v14/config/cache_config/cache_config.go

```diff
@@ -0,0 +1,265 @@
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cache_config
+
+import (
+	"regexp"
+	"strings"
+
+	. "github.com/apache/trafficcontrol/traffic_ops/testing/api/v14/config"
```

Comment:
Anonymous imports should be avoided in Go. It makes it unclear where symbols are coming from. The Go language itself recommends against it: https://github.com/golang/go/wiki/CodeReviewComments#import-dot

You might consider renaming it `cfg` to be shorter, too. I generally prefer the shortest unambiguous abbreviation, to make package easier to use.

Reply:
I was aware that the use of the dot import should be limited, but it seemed suitable due to the file structure, and how every dot importer _is a_ config parser. I didn't use the dot import because I thought 'config' was too long.

I can still change it though.